compiler/parser/parser.peg

```diff
@@ -148,7 +148,7 @@ SearchOrTerm = _ OrToken _ t:SearchAnd { RETURN(ARRAY("or", t)) }
 
 SearchAnd
   = first:SearchFactor
-    rest:((_ AndToken)? _ expr:SearchFactor{ RETURN(ARRAY("and", expr)) })* {
+    rest:(!(_ OrToken _) (_ AndToken)? _ expr:SearchFactor{ RETURN(ARRAY("and", expr)) })* {
```

Comment:
```suggestion
    rest:((_ AndToken)? _ !OrToken expr:SearchFactor{ RETURN(ARRAY("and", expr)) })* {
```

Reply:
Wouldn't this mean that you could have a search like: `x>1 order>1`?

Reply:
Tests passed with this change so I figured that meant it was a winner.

Reply:
Nevermind, looks like you are right.